### PR TITLE
Re-depend on GNOME schema for date and time format

### PR DESCRIPTION
**Pull request by tintou**
All the features are still there.

### CODE REVIEW

**Comment by jeremypw** [comment]
Maybe problem with switchboard plug if this is merged, so blocking for now.

File: data/io.elementary.desktop.wingpanel.datetime.gschema.xml [hunk 1]
```diff
@@ -1,35 +1,15 @@
...
-    <value nick="24h" value="1" />
-    <value nick="ISO8601" value="2" />
-  </enum>
     <schema path="/io/elementary/desktop/wingpanel/datetime/" id="io.elementary.desktop.wingpanel.datetime">
         <key type="b" name="show-weeks">
             <default>false</default>
             <summary>Show week numbers.</summary>
             <description>If this is set to true, week numbers will be shown in the calendar.</description>
         </key>
-        <key name="clock-format" enum="io.elementary.desktop.wingpanel.datetime.clock-formats">
```
Comment (jeremypw) [inline data/io.elementary.desktop.wingpanel.datetime.gschema.xml]: Deleting this key breaks Granite.DateTime.is_clock_format_12h.

Comment (tintou) [reply data/io.elementary.desktop.wingpanel.datetime.gschema.xml]: Yes, it should be merged with https://github.com/elementary/granite/pull/273

**Comment by danrabbit** [comment]
Gonna close this since we merged #147

**Pull request closed without merging**